src/config/hardware.h

```diff
@@ -10,14 +10,16 @@
 #define POT_MAX 1023
 
 // Pins //
-// Interface //
+// Interrupts //
```

Comment:
In the current scheme (changes pending with new Board abstraction) these are organized by their purpose -- so "Interface" is for things that interface with the pattern switcher and "Inputs" is for things that act as readable inputs for patterns.